lib/amazon/ecs.rb

```diff
@@ -137,10 +137,11 @@ def self.send_request(opts)
       log "Request URL: #{request_url}"
 
       res = Net::HTTP.get_response(URI::parse(request_url))
+      body = Response.new(res.body)
       unless res.kind_of? Net::HTTPSuccess
-        raise Amazon::RequestError, "HTTP Response: #{res.code} #{res.message}"
+        raise Amazon::RequestError, "HTTP Response: #{res.code} #{res.message}#{ body.error}"
```

Comment:
That's a great patch btw, thanks, could we improve on the variable naming and the formatting a bit? Something like the snippet below, thanks!
```ruby
      ecs_res = Response.new(res.body)
      unless res.kind_of? Net::HTTPSuccess
        err_msg = "HTTP Response: #{res.code} #{res.message}"
        err_msg += " - #{ecs_res.error}" if ecs_res.error
        raise Amazon::RequestError, err_msg
      end
      ecs_res
```